tests/data/procurve.json

```diff
@@ -549,7 +549,7 @@
                     "sysDescr": "HP Stack, revision KA.15.03.3004, ROM KA.15.05 (/sw/code/build/tam(KA_15_03))",
                     "sysContact": "<private>",
                     "version": "KA.15.03.3004",
-                    "hardware": "",
+                    "hardware": "Stack",
```

Comment:
This seems wrong

Reply:
I thought it was more right than "".  This is also something that could be fixed by `ENTITY-MIB::entPhysicalDescr.1` maybe.